assets/app/views/create/fromimage.html

```diff
@@ -121,6 +123,18 @@
                               <i class="pficon pficon-help" data-toggle="tooltip" data-placement="right" data-original-title="Additional routes with alternative configuration options (security, TLS termination, etc) can be added using the oc command."></i>
                             </span>
                         </label>
+
+                      </div>
+                      <div class="form-group">
+                        <label for="routeTargetPort">Target port:</label>
+                        <span ng-if="routing.portOptions.length === 1">{{routing.portOptions[0].containerPort}}/{{routing.portOptions[0].protocol}}</span>
+                        <select
+                            id="routeTargetPort"
+                            ng-if="routing.portOptions.length > 1"
+                            ng-model="routing.targetPort"
+                            ng-options="portSpec.containerPort + '/' + portSpec.protocol for portSpec in routing.portOptions"
```

Comment:
`(portSpec.containerPort + '/' + portSpec.protocol) for portSpec in routing.portOptions` ?